Gems/ScriptCanvas/Code/Editor/Framework/Configuration.cpp

```diff
@@ -23,6 +23,8 @@
 
 namespace ScriptCanvasEditor
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether